libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -47,7 +47,7 @@
 </ng-container>
 <ng-template #empty>
   <div class="empty">
-    <p>
+    <p id="searchTip">
```

Comment:
This doesn't seem to be referenced anywhere else. Generally if you're going to use ids, don't use camelCase, use kebab-case. If this is for testing, you want to make it `data-testid="..."`

Reply:
Hint: is the usage of an anchor tag on line 51 appropriate? What should it be instead?

Reply:
reverted